otterdog/eclipse-symphony.jsonnet

```
@@ -47,6 +47,7 @@ orgs.newOrg('iot.symphony', 'eclipse-symphony') {
           requires_conversation_resolution: true,
           bypass_pull_request_allowances+: [
             "@eclipse-symphony-bot",
+            "user:Haishi2016",
```

Actor must be referenced as described in https://otterdog.readthedocs.io/en/latest/reference/organization/repository/actor/

```suggestion
            "@Haishi2016",
```

---

Updated